node-operators/block-producers/offboarding/account.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Decommission Block Producer
+description: In these step-by-step instructions, learn how to properly offboard as a Tanssi block producer, including unmapping your session keys and unstaking your bond.
+---
+
+# Offboard as a Tanssi Block Producer
+
+## Introduction {: #introduction }
+
+This guide will show you how to wind down your operations as a Tanssi block producer properly. Naturally, this guide assumes that you're an existing Tanssi block producer with a [block-producing node](/node-operators/block-producers/onboarding/run-a-block-producer){target=\_blank} and mapped [session keys](https://wiki.polkadot.network/docs/learn-keys#session-keys){target=\_blank}. Offboarding a block producer node is a two-step process that involves removing your delegation and unmapping your session keys. Following the steps outlined in this guide, you can cease operations as a Tanssi block producer. 
+
+## Request Undelegation {: #request-undelegation }
+
+When you set up your Tanssi block producer node, you had to submit a delegation bond of at least `{{ networks.dancebox.block_producers.min_self_del.dance }}` DANCE. To get that back and remove your block producer from the list of eligible candidates, you'll need to take steps similar to those in the onboarding process.
+
+### Viewing Existing Stake {: #viewing-existing-stake }
+
+Before undelegating, it is helpful first to see how much you have staked, as you'll need to provide this figure later. To do so, head to [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss://fraa-dancebox-rpc.a.dancebox.tanssi.network#/chainstate){target=\_blank}, click on the **Developer** tab, select **Chain State** from the dropdown, and take the following steps:
+
+1. Select the **pooledStaking** module
+2. Select the **pools** query
+3. Enter your block producer account
+4. Ensure that the **include option** slider is toggled on
+5. In the **option** field, you can select **CandidateTotalStake** to see the amount you have staked in Plank. Alternatively, you can select **AutoCompoundingShares** or **ManualRewardShares** depending on the staking configuration used when setting up your block producer node 
```

Comment:
i misled you on this; i'm sorry! 😅 😬  - its not `CandidateTotalStake`. I'm making sure I 100% understand this before getting back to you

Reply:
gahhh, so there isn't a super straightforward way to do this with polkadot.js apps right now. users can query each of the following and add them: ManualRewardShares + AutoCompoundingShares + JoiningShares

so maybe for this guide, for step 5 use JoiiningShares and then after the image just tell them to repeat the steps and query ManualRewardShares and AutoCompoundingShares and add em up to get their self-del?

Reply:
CandidateTotalStake includes their entire stake for all their delegations (not just their own) - shocking, i know! 🙃  haha i don't know how i made that mistake initially, my bad